snowflake/JsonUploadFromBlob.txt

```diff
@@ -0,0 +1,123 @@
+-- SELECT DATABASE --
+use database BLOB_DB;
+
+use schema PUBLIC;
+
+use role accountadmin;
+
+-- Create stage --
+
+create or replace stage azure_stage_sas
+  URL='azure://<storage_account>.blob.core.windows.net/<container>'
+  CREDENTIALS=(AZURE_SAS_TOKEN='?<sas-token>');
+
+list @azure_stage_sas;
+
+-- Import raw data to table --
+
+create or replace table RAW_SOURCE (
+  SRC VARIANT);
+
+create or replace task copy_data
+warehouse=PLAYWH
+schedule='1 MINUTE'
+AS
+copy into RAW_SOURCE from @azure_stage_sas
+FILE_FORMAT=(TYPE=json);
+
+alter task copy_data resume;
+
+-- Create source table with hash--
+
+create or replace table RAW_SOURCE_HASHED as select * from RAW_SOURCE;

```

Comment:
We don't need to have two tables `RAW_SOURCE` and `RAW_SOURCE_HASHED`. Copying the data and computing hash can (and should) be done in one task.